### PR TITLE
Editorial: thank wide review groups in acknowledgements

### LINKED ISSUE #265: No explicit acknowledgements?
Looking through this repo makes it seem like the acknowledgements for this spec are rather miserly, ignoring groups of people like PING as if they made no contribution. That might be common practice in the Web Apps group now (there are different practices across different groups and across time), but it seems unnecessarily exclusive.

🤷

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #265

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/web-share/pull/269.html" title="Last updated on Jan 24, 2023, 1:09 AM UTC (032a4e5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-share/269/5c9977c...032a4e5.html" title="Last updated on Jan 24, 2023, 1:09 AM UTC (032a4e5)">Diff</a>

### CODE REVIEW

**Review by ewilligers — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -765,6 +765,17 @@ <h2>
       <h2>
         Acknowledgments
       </h2>
+      <p>
+        The editors would like to thank the W3C's horizontal review groups for
```
Comment (chaals) [inline index.html:769]:
```suggestion
        The editors would like to thank the following W3C groups for
```

File: index.html [hunk 2]
```diff
@@ -765,6 +765,17 @@ <h2>
...
+        their invaluable feedback, which greatly improved this specification.
```
Comment (chaals) [inline index.html:770]:
```suggestion
        their invaluable feedback, which greatly improved this specification:
```

File: index.html [hunk 3]
```diff
@@ -765,6 +765,17 @@ <h2>
...
+        Namely, we'd like to thank the <a href=
```
Comment (chaals) [inline index.html:771]:
```suggestion
        <a href=
```

**Review by chaals — APPROVED** [review]
This is a net improvement whether or not you adopt my prose.

**Review by ewilligers — APPROVED** [review]

**Pull request merged**